uplink/utils.py

```diff
@@ -58,6 +59,19 @@ def get_arg_spec(f):
 except ImportError:
     import urlparse
 
+try:
+    from functools import lru_cache as memoize
+except ImportError:
```

Comment:
Looks like the coveralls failure is strictly due to a loss of coverage in this file. Given that it's pretty straightforward memoization, I think it's fair to have `coverage.py` skip this block. Could you add this coverage add  `# pragma: no cover` to this line?

Reply:
Or why not just add a test? ;)